lib/rufo/command.rb

```diff
@@ -176,12 +188,16 @@ def self.parse_options(argv)
         filename_for_dot_rufo = value
       end
 
+      opts.on("-x", "--no_exit_code", "Return 1 in the case of failure, else 0") do
```

Comment:
Can we do dashes instead of underscores here? That seems much more standard for CLI options

Reply:
Good catch - that was a mistake (see title) 😊